compute/cloud-client/src/test/java/compute/windows/osimage/WindowsOsImageIT.java

```diff
@@ -151,27 +147,27 @@ public void beforeEach() {
 
   @AfterEach
   public void afterEach() {
-    stdOut = null;
-    System.setOut(null);
+    System.setOut(System.out);
   }
 
   @Test
-  public void testCreateWindowsImage_failDueToRunningInstance()
+  public void testCanCreateImage()
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
-    Assertions.assertThrows(
-        IllegalStateException.class,
-        () -> CreateImage.createImage(PROJECT_ID, ZONE, DISK_NAME, IMAGE_NAME,
-            "eu", false),
-        String.format("Instance %s should be stopped.", INSTANCE_NAME));
+    CreateImage.createImage(
+        PROJECT_ID, ZONE, getBootDiskName(testInstanceName), testImageName, "us", true);
```

Comment:
Should we make "us" a constant? Previously this was set to "eu" and it may be helpful to flag that more clearly in the sample.

Reply:
It is set to `"us"` only to speed up the process because our tests run in the `us-central1` region. Since it is used only once I think that converting it to a constant can be postponed.